tests/test_grid3d/test_grid_property.py

```diff
@@ -697,3 +696,48 @@ def test_gridprop_export_bgrdecl_double(tmp_path, grid):
 
     with open(tmp_path / "actnum.DATA", "rb") as fh:
         assert b"DOUB" in fh.read()
+
+
+@pytest.mark.parametrize(
+    "simcase, props, expected_avg, description",
+    [
+        (
+            "E100_BO",
+            ["SOIL", "PRESSURE"],
+            [0.0119888, 270.03588],
+            "E100 blackoil",
+        ),
+        (
+            "E300_BO",
+            ["SOIL", "PRESSURE"],
+            [0.0119888, 270.03588],
+            "E300 blackoil",
+        ),
+        (
+            "E300_COMP",
+            ["SOIL", "PRESSURE"],
+            [0.00945095, 400.720811],
+            "E300 composite",
+        ),
+    ],
+)
+def test_simpleb8_case(simcase, props, expected_avg, description):
+    """Test a very small case that is actually ran through E100, E300, IX."""
+
+    folder = TPATH / "3dgrids" / "simpleb8"
+
+    grd = xtgeo.grid_from_file(folder / str(simcase + ".EGRID"), fformat="egrid")
+
+    assert grd.dimensions == (4, 2, 3)
```

Comment:
This is quite a small case, is it possible to create it from scratch instead of reading from file?

Reply:
No, then we would not catch how the various simulator flavors actually output formats that xtgeo is able to read. E.g. the IX EGRID/UNRST have issues that we struggle with still.